CryptoSwift/MD5.swift

```diff
@@ -107,11 +107,11 @@ final class MD5 : HashProtocol  {
             hh[3] = hh[3] &+ D
         }
 
-        let buf: NSMutableData = NSMutableData();
-        hh.map({ (item) -> () in
+        let buf: NSMutableData = NSMutableData()
+        hh.forEach { item in
```

Comment:
Nope, I changed it myself. There was a warning that the result of map wasn't used.